php-checks/src/main/java/org/sonar/php/checks/UselessExpressionStatementCheck.java

```diff
@@ -104,4 +114,18 @@ public void visitToken(SyntaxToken token) {
       fileContainsHTML = true;
     }
   }
+
+  private Stream<Pattern> regularStringExceptionsPatterns() {
```

Comment:
I think that we make things more complex that needed. Maybe in the future we well need to handle more complex exceptions for string literals, but for the moment it's simple:
* there's only one pattern to check
* we know this pattern at compile time

So, I think that we can simply define a constant in this class like:
```
private static final Pattern STRING_LITERAL_EXCEPTION_PATTERN = Pattern.compile("@phan-var");
```